### PR TITLE
[Docs] XMLRPC - Repair Syntax, Params, Descriptions and Refs

### PR DESCRIPTION
**Pull request by seaniepie**
All unified and fixed

### CODE REVIEW

File: docs/dictionary/function/revXMLRPC_GetPath.lcdoc [hunk 1]
```diff
@@ -18,40 +18,41 @@ Platforms: desktop, server, mobile
...
+put revXMLRPC_GetPath(tRequest) into tPath
 
 Parameters:
-documentID:
-The number returned by the revXMLRPC_CreateRequest when you created the
-XML-RPC request, or the successful revXMLRPC_Execute function when you
-executed an XML-RPC request.
+XMLRPCdocumentID (integer):
+The number returned by the <revXMLRPC_CreateRequest> when you created the
+<XML-RPC> request, or the successful <revXMLRPC_Execute< <function> when you
```
Comment (peter-b) [inline docs/dictionary/function/revXMLRPC_GetPath.lcdoc]: `<revXMLRPC_Execute<` should be `<revXMLRPC_Execute>`.

File: docs/dictionary/function/revXMLRPC_GetHost.lcdoc [hunk 2]
```diff
@@ -18,39 +18,40 @@ Platforms: desktop, server, mobile
...
-> <standalone application>, you must include this 
-> <LiveCode custom library|custom library> when you create your 
-> <standalone application|standalone>. In the Inclusions pane of the 
-> <Standalone Application Settings> window, make sure the "XML-RPC" 
+>*Important:*  The <revXMLRPC_GetHost> <function> is part of the
+> <XML-RPC library>. To ensure that the <function> works in a
+> <standalone application>, you must include this
+> <LiveCode custom library|custom library> when you create your
+> <standalone application|standalone>. In the Inclusions pane of the
+> <Standalone Application Settings> window, make sure the "XML-RPC"
```
Comment (peter-b) [inline docs/dictionary/function/revXMLRPC_GetHost.lcdoc]: I note that only a minority of the XML-RPC library syntax dictionary entries have this "Important" paragraph.  However, I'm not sure it's ideal to copy and paste it into every dictionary entry related to the XML-RPC library.  Do you think there's any logical place to just put the warning once, but where most people using the library are likely to see it?  Is there an overall summary page for the XML-RPC library that might be a good place to stash it?

**Review by peter-b — COMMENTED** [review]
I just noticed a couple of issues while looking through this.  Big improvement overall, though!

**Comment by seaniepie** [comment]
The most obvious place would be to add it into the revXMLRPC_CreateRequest as this will be the first place  to visit for any first time XMLRPC user. After that there are a few other key one like revXMLRPC_Execute, revXMLRPC_GetRequest, revXMLRPC_GetResponse, revXMLRPC_SetSocket and perhaps revXMLRPC_DeleteAllDocuments and revXMLRPC_DeleteDocument. Then the glossary for 'XML-RPC' and 'XML-RPC document'

**Comment by peter-b** [comment]
@seaniepie That sounds like a sensible approach. Do you think it would be best to make sure that the inclusion note is in all the right places as part of this PR, or maybe as a separate one?

**Comment by seaniepie** [comment]
Lets go for it. I think it would be nice to have this done _and_ dusted. With that in mind does revXMLRPC_DeleteParam need a result value? Does it also respond with the err if there is one?

I'll make and add in to this repo the changes to the glossary items.

**Comment by peter-b** [comment]
From the [source code](https://github.com/livecode/livecode-ide/blob/5b6cb938ae716e1c691d3876ce4eee6b04bfecc8/Toolset/libraries/revxmlrpclibrary.livecodescript#L146) it appears that `revXMLRPC_DeleteParam` sets the result to an error message if it fails to delete the parameter.

**Comment by seaniepie** [comment]
I'm making sure that a link to the XML-RPC library (library) is included in every doc too.

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 371d684

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 371d684c074a63485dbb95a27b32704b4cc11335

**Comment by livecode-vulcan** [comment]
😎 test success 371d684c074a63485dbb95a27b32704b4cc11335
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/542)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/547)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/542)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/546)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/551)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/543)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/543)

**Pull request merged**